Create ui-c8y-1021-0-0-upgrade-to-angular-18-6833.md (#2582)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1021-0-0-upgrade-to-angular-18-6833.md

---
date: ""

> title: Upgrade to Angular 18 (#6833)

Author: The Claude AI suggested to use the following title:
```suggestion
title: Upgrade to Angular 18
```

product_area: Application enablement & solutions
change_type:
  - value: change-QHu1GdukP
    label: Feature
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-60178
version: 1021.0.0
---

> Upgrade to Angular 18 (#6833)

Author: The Claude AI suggested to use the following body:
```suggestion
As part of keeping the Cumulocity IoT platform up-to-date with the latest web technologies, Cumulocity IoT has been upgraded to use Angular 18. This upgrade brings the latest features, performance improvements, and bug fixes from the Angular framework to the Cumulocity IoT platform. The upgrade to Angular 18 should be largely transparent to users, with no visible changes to the user interface or existing functionality. However, any custom widgets or plugins developed for previous versions of Cumulocity IoT may need to be updated to ensure compatibility with Angular 18.
```

> As part of keeping the Cumulocity IoT platform up-to-date with the latest web technologies, Cumulocity IoT has been upgraded to use Angular 18. This upgrade brings the latest features, performance improvements, and bug fixes from the Angular framework to the Cumulocity IoT platform. The upgrade to Angular 18 should be largely transparent to users, with no visible changes to the user interface or existing functionality. However, any custom widgets or plugins developed for previous versions of Cumulocity IoT may need to be updated to ensure compatibility with Angular 18.

Reviewer: ```suggestion
As part of keeping the {{< product-c8y-iot >}} platform up-to-date with the latest web technologies, {{< product-c8y-iot >}} has been upgraded to use Angular 18. This upgrade brings the latest features, performance improvements, and bug fixes from the Angular framework to the {{< product-c8y-iot >}} platform. The upgrade to Angular 18 should be largely transparent to users, with no visible changes to the user interface or existing functionality. However, any custom widgets or plugins developed for previous versions of {{< product-c8y-iot >}} may need to be updated to ensure compatibility with Angular 18.
```
